### ISSUE TITLE
ledFace animations glitch out after a while

### ISSUE THREAD

**Issue by kefcom** [Brixel/fri3dbot#3]
It seems ledFace script isn't working properly. the mood times start to override each other after a while.
To be decided if fix is needed, possibly we scrap the whole face, it's the worst one anyway.

**Comment by kefcom** [comment]
Logfile showing duplication of events after robotscript died. will investigate more.
I suspect this duplication happens multiple times during the display of this face, so the scenes get loaded at random times by different scripts, causing the glitching effect.
![image](https://user-images.githubusercontent.com/6786386/39579057-19084142-4ee6-11e8-8cad-8cc551170b33.png)

**Comment by kefcom** [comment]
So it seems there were 2 different problems.
1: checking if scene begins with ledFace_Robot is a bad idea when all the scenes for this mood begin with ledFace_robot. So I changed the scene names to be more consistent with the naming we use in other scenes.

2: When the new mood id is 4 (sleeping) or 6 (party), the script will check if this mood is allowed (only allowed at certain times). Foolish me for thinking that when I called the determinemood function would be enough to exit the routine :(. so I added a return; after each check so the script doesn't pass the end of the changemood() routine twice (or more). I mean... it's my first c# experience ;).

after a little testing it seems the issue is resolved by this fix.
